Candidatura/Preventivo dei costi/Preventivo dei costi.typ

```diff
@@ -0,0 +1,92 @@
+
+==  Periodo di sviluppo della Requirements and Technologies Baseline
+A seguito del risultato prodotto nella fase precedente di analisi dei requisiti, ampio
+spazio sarà dedicato alla progettazione e all’utilizzo pratico delle tecnologie necessarie
+allo sviluppo, dimostrando la loro validità e funzionalità.
+Il prodotto finale di questa fase sarà il Proof of Concept dimostrante la concreta possibilità di poter utilizzare le tecnologie in modo cooperante tra loro.
+
+== Periodo di sviluppo del Minimun Viable Product (MVP)
+Dimostrata la concreta possibilità implementativa mediante il Proof of Concept, il gruppo
+impiegherà le risorse nella realizzazione materiale di un prodotto software atto al soddisfacimento almeno dei requisiti minimi di accettazione, fornendo tutte le funzionalità
+richieste per essere considerato valido, ponendo attenzione non solo al lato funzionale
+ma anche, e soprattutto, al lato implementativo.
+
+= Analisi dei ruoli
+
+== Responsabile
+Il responsabile è un ruolo presente durante l'intero progetto. Il suo compito è quello di gestire il team di assicurarsi che lo sviluppo proceda secondo le tempistiche predefinite e le aspettative del committente. Deve inoltre redigere e far rispettare le norme di progetto, nonché le linee guida di sviluppo che l'intero team deve rispettare. Essendo il ruolo più costoso, il numero di ore è stato scelto per favorire l'efficienza e non pesare eccessivamente sul costo finale.
+
+== Amministratore
+L'amministratore è un ruolo presente durante l'intero progetto, in quanto si occupa di predisporre e controllare il corretto utilizzo delle procedure e degli strumenti definiti nel _way of working_, andando anche a gestire e implementare automatismi, migliorando così l'efficienza del team. Il monte ore scelto è tale, poiché essendo questo un ruolo di controllo, non sono richieste un numero elevato di ore.
+
+== Analista
+L'analista è presente esclusivamente nelle fasi iniziali del progetto, poiché si occupa della raccolta e dell'elaborazione dei requisiti. Ciò comporta un numero di ore ridotto e una distribuzione delle stesse unicamente nelle prime settimane del progetto.
+
+== Progettista
+Il progettista ha il compito di delineare e documentare l'architettura del prodotto in modo da:
+
+- soddisfare i requisiti raccolti nelle fasi pregresse;
+
+- aiutare il team di sviluppo con una documentazione chiara ed esaustiva.
+
+Nello stabilire l'architettura deve quindi indicare anche quali saranno le tecnologie da utilizzare per la sua implementazione.
+Ritenendolo un ruolo impegnativo dal punto di vista temporale, il numero di ore risulta maggiore rispetto ai ruoli precedenti.
+
+== Programmatore
+Il programmatore ha il compito di sviluppare concretamente l'architettura prodotta dal progettista, traducendola in codice sorgente. Il ruolo prevede un numero di ore molto elevato poiché riteniamo il lavoro più dispendioso rispetto a quello delle altre figure professionali.
```

Comment:
Non "sviluppare" ma "tradurre in codice eseguibile" o qualcosa del genere. Lo sviluppo appartiene ad altri ruoli.
Più che "dispendioso" direi che richiede tante ore.